packages/checkout-ui-extensions/src/components/Stepper/Stepper.ts

```diff
@@ -1,31 +1,41 @@
 import {createRemoteComponent} from '@remote-ui/core';
 
-import {TextFieldProps} from '../TextField';
 import {IconSource} from '../Icon';
 
-export interface StepperProps
-  extends Pick<
-    TextFieldProps<number | undefined>,
-    | 'id'
-    | 'name'
-    | 'label'
-    | 'prefix'
-    | 'accessibilityDescription'
-    | 'required'
-    | 'error'
-    | 'disabled'
-    | 'readonly'
-    | 'value'
-    | 'onFocus'
-    | 'onBlur'
-    | 'onInput'
-    | 'onChange'
-  > {
+export interface StepperProps {
```

Comment:
@oliverigor Do you know if we replaced the `extends` for docs generation purposes?

Reply:
The problem here was that the `Pick` typescript method was noth working for the docs generation so I changed where needed in [this PR](https://github.com/Shopify/checkout-web/pull/19302) and filed [this issue](https://github.com/Shopify/generate-docs/issues/159) on their repo